index.js

```diff
@@ -12,67 +12,83 @@ function LessCompiler(sourceNodes, inputFile, outputFile, _options) {
     return new LessCompiler(sourceNodes, inputFile, outputFile, _options);
   }
 
-  CachingWriter.call(this, Array.isArray(sourceNodes) ? sourceNodes : [sourceNodes], _options);
+  CachingWriter.call(
+    this,
+    Array.isArray(sourceNodes) ? sourceNodes : [sourceNodes],
+    _options
+  );
+
+  if (!outputFile) {
```

Comment:
If not outputFile is specified I now default it to the input path with a css ext